### DISCUSSION TITLE
API endpoint

### DISCUSSION THREAD

**Discussion by TiwariAbhishek23** [forem/forem#18900] [Q&A]
Hello, I am working on my portfolio and want to display all my posts, and I am able to display that. Still, I also want that whenever a user clicks on the post the user should not get redirected to dev.to but read the article on my website, I am looking for such an endpoint but didn't find, as per chatGPT there is one such endpoint https://dev.to/api/articles/${id} like this, but I am getting 404 error. Please guide me.
Thanks

**Comment by thomasbnt** [comment]
Hello ! 

You have the documentation at https://developers.forem.com/, the v0 will be depreciated, but you can use it for the moment. 
Forem team works on the v1. 

If you want an example, I created a simple webpage with the API v0. 

https://devtoprofile.thomasbnt.dev/

> user clicks on the post the user should not get redirected to dev.to but read the article on my website

For that, maybe create a route /article/{slug} on your app, with that, you can show an DEV article on your website. It's just an idea
